src/legacy/core_plugins/kibana/public/management/sections/objects/components/objects_table/components/relationships/relationships.js

```diff
@@ -227,7 +227,7 @@ class RelationshipsUI extends Component {
               items={list}
               columns={[
                 {
-                  width: 24,
+                  width: '24px',
```

Comment:
All the snapshot updates: are these just maintenance needed because these tests never ran previously?

Reply:
It looks like this change and the snapshot changes should not be in this PR